styles.css

```diff
@@ -0,0 +1,53 @@
+*{
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: rgb(233, 224, 224);
+}
+
+#main{
+  text-align: center;
+}
+
+#title{
+ 
+  
+}
```

Comment:
- It would be good not to have an empty ID like this without good reason as it is best practice.
- You could add any necessary styling to it or delete this empty ID as having it like this serves no purpose to the HTML file.